contact-widgets.php

```diff
@@ -70,6 +79,19 @@ public function __construct( $cur_php_version = PHP_VERSION ) {
 		}
 
 		/**
+		 * Setup the Front Awesome assets URL
+		 *
+		 * @return string Returns URL from where Font Awesome should load from.
+		 *
+		 * @since NEXT
+		 */
+		public function font_awesome_url() {
+
+			return ! apply_filters( 'wpcw_widget_social_icons_use_cdn', false ) ? static::$assets_url . 'css/font-awesome.min.css' : apply_filters( 'wpcw_widget_social_icons_cdn_url', '//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css' );
```

Comment:
@EvanHerman A few things:

1. Break these filters into their own lines and give them each a docblock.
2. Cast the expected value type for each.
3. Whether using bundled FA or the CDN, we should only display minified when `SCRIPT_DEBUG` is false.